helm-chart/templates/deployment.yaml

```diff
@@ -68,18 +70,32 @@ spec:
               value: {{ .Values.environment.db_pwd}}
           ports:
             - name: http
-              containerPort: 80
+              containerPort: {{ .Values.container.port }}
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /webservices/status.php
+              path: /setup/
               port: http
           readinessProbe:
             httpGet:
-              path: /webservices/status.php
+              path: /setup/
               port: http
```

Comment:
Why this change? The status should report if iTop is healthy.

Reply:
I don't know why but this page sent 500 error.

Reply:
Maybe better to find that out.

Reply:
What is the message the page is sending or what do you see in the logs?

Reply:
I have 500, I was searching about documentation to know if that page is available before first setup. 
Because now all is ok with that file. 
`curl localhost:8080/webservices/status.php
{"status":"RUNNING","code":0,"message":""}`

Reply:
The page is indeed giving php error when iTop is not yet installed..

Reply:
And these PHP error send wich kind of http error ? 500 ?

Reply:
Hmm, no it was a `200 OK` 🤔

Reply:
I know what is the problem, the status.php return 500 because the config-itop.php is missing. 
That file is missing because in kube, when you mount a volume, kubernetes will mount empty directory in `/var/www/html/conf` so the directory with config-itop.php is overwrite by an empty directory

Reply:
Is that also related to the change you made in `deployment.yaml`?

Reply:
No it's diffrent issue. This one will need many changes. 
I'll do tests on my repo and I'll coming back with another pull request for this point. It's ok for you ?

Reply:
Roll back, I'll open another PR with that modification